brokers/unified/cmd/main.go

```diff
@@ -34,6 +36,16 @@ func main() {
 		broker.PushEvents(statusTun)
 	}
 
-	metas := cfg.ReadConfig()
+	var metas []model.PluginMeta
+	if cfg.DownloadMetas {
+		pluginFQNs := cfg.ParsePluginFQNs()
+		pluginMetas, err := utils.New().GetPluginMetas(pluginFQNs, cfg.RegistryAddress)
+		if err != nil {
+			log.Fatalf("Failed to download plugin metas: %s", err)
```

Comment:
Broker has special method for this purpose that insures that error message is sent to WS master.

Reply:
I didn't realize -- done.